tests/MongoDBDriverManager_executeBulkWrite-error-001.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+MongoDB\Driver\Manager::executeBulkWrite() update write error
+--FILE--
+<?php
+include dirname(__FILE__) . '/utils.inc';
+
+$m = new MongoDB\Driver\Manager("mongodb://localhost:27017");
+cleanup( $m );
+
+$bw = new MongoDB\Driver\BulkWrite( [ 'ordered' => false ] );
```

Comment:
Is `[ 'ordered' => false ]` relevant? I'd think to just leave the ctor arg empty for conciseness.